www/templates/html/MediaPlayer.tpl.php

```diff
@@ -6,13 +6,184 @@
 }
 ?>
 
+<?php if($context->media->getTextTracks()): ?>
```

Comment:
We'll probably want to adjust this class method so that it caches the return value from the Amara API, since we'll be calling it a couple of times now, we want to avoid going back to the API during the same request.